roachtest: pgx failed

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1665474&tab=buildLog) on [release-19.2@8ca08b3ddbfa07eb8fc19a1035cf82539f05c7a3](https://github.com/cockroachdb/cockroach/commits/8ca08b3ddbfa07eb8fc19a1035cf82539f05c7a3):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191228-1665474/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-35-g8ca08b3
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist19_2) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1665474&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1665456&tab=buildLog) on [master@8f946b8bb62629002c5e958373b81ca9e1920dd6](https://github.com/cockroachdb/cockroach/commits/8f946b8bb62629002c5e958373b81ca9e1920dd6):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191228-1665456/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-933-g8f946b8
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1665456&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1666388&tab=buildLog) on [master@edd46b1405b7928760b65ec8aad59fd22a8adb6b](https://github.com/cockroachdb/cockroach/commits/edd46b1405b7928760b65ec8aad59fd22a8adb6b):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191229-1666388/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-938-gedd46b1
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1666388&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668189&tab=buildLog) on [master@f1f15250871d124178f40bf340ef48719b50e7e5](https://github.com/cockroachdb/cockroach/commits/f1f15250871d124178f40bf340ef48719b50e7e5):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191231-1668189/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-942-gf1f1525
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668189&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668207&tab=buildLog) on [release-19.2@3960c23e7b8fef3042d525f74768c5ae85a1cf22](https://github.com/cockroachdb/cockroach/commits/3960c23e7b8fef3042d525f74768c5ae85a1cf22):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191231-1668207/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-37-g3960c23
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist19_2) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668207&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1669512&tab=buildLog) on [master@0d1ae9cbc49518180e68aa0bfbf982d73854ddae](https://github.com/cockroachdb/cockroach/commits/0d1ae9cbc49518180e68aa0bfbf982d73854ddae):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200101-1669512/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-945-g0d1ae9c
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1669512&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1670273&tab=buildLog) on [master@a99740bb34c03ab3af7558129edce9c17c88d4f9](https://github.com/cockroachdb/cockroach/commits/a99740bb34c03ab3af7558129edce9c17c88d4f9):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200102-1670273/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-947-ga99740b
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1670273&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671769&tab=buildLog) on [release-19.2@9aaf5fb6f7108858d200dd3d199deb11871a09b4](https://github.com/cockroachdb/cockroach/commits/9aaf5fb6f7108858d200dd3d199deb11871a09b4):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200103-1671769/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-44-g9aaf5fb
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist19_2) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671769&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671733&tab=buildLog) on [master@8facb469d66426a3f372a1aed4ea6f34e287b501](https://github.com/cockroachdb/cockroach/commits/8facb469d66426a3f372a1aed4ea6f34e287b501):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200103-1671733/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-981-g8facb46
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671733&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1673622&tab=buildLog) on [master@7906cbe3d9c615430f6e892a42e6bccdad2aff00](https://github.com/cockroachdb/cockroach/commits/7906cbe3d9c615430f6e892a42e6bccdad2aff00):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200104-1673622/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-995-g7906cbe
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1673622&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1673640&tab=buildLog) on [release-19.2@6cc3423103f85d7f82f01791a24d99bef21b2546](https://github.com/cockroachdb/cockroach/commits/6cc3423103f85d7f82f01791a24d99bef21b2546):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200104-1673640/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-47-g6cc3423
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist19_2) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1673640&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1674751&tab=buildLog) on [master@f180501b7c2ccb946d7f7c8944d5509b87cb0394](https://github.com/cockroachdb/cockroach/commits/f180501b7c2ccb946d7f7c8944d5509b87cb0394):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200106-1674751/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1001-gf180501
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1674751&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676232&tab=buildLog) on [43008@b555d16b4455f7122f4211082512820742f8116f](https://github.com/cockroachdb/cockroach/commits/b555d16b4455f7122f4211082512820742f8116f):

```
The test failed on branch=43008, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200106-1676232/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1011-gb555d16
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676232&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676733&tab=buildLog) on [release-19.2@f3bcbd305602c89d52cb7ea4cb114f362c5aee23](https://github.com/cockroachdb/cockroach/commits/f3bcbd305602c89d52cb7ea4cb114f362c5aee23):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200107-1676733/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-49-gf3bcbd3
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist19_2) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676733&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676715&tab=buildLog) on [master@52261c39c65aecf960e3c8d54089a66d378af610](https://github.com/cockroachdb/cockroach/commits/52261c39c65aecf960e3c8d54089a66d378af610):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200107-1676715/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1030-gd09fd7f
		Tests run against pgx v4.1.2
		116 Total Tests Run
		61 tests passed
		55 tests failed
		0 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676715&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1688037&tab=buildLog) on [release-19.2@7f2413b36fe1ea31a08c3ddd0d1e4b1cd006db37](https://github.com/cockroachdb/cockroach/commits/7f2413b36fe1ea31a08c3ddd0d1e4b1cd006db37):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200114-1688037/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-72-g7f2413b
		Tests run against pgx v4.2.1
		116 Total Tests Run
		62 tests passed
		54 tests failed
		0 tests skipped
		2 tests ignored
		1 test passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: v4.TestArrayDecoding - https://github.com/cockroachdb/cockroach/issues/26925 (unexpected)
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist19_2) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1688037&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1688019&tab=buildLog) on [master@3d98dae853edbde083cc23b60ad6696958909849](https://github.com/cockroachdb/cockroach/commits/3d98dae853edbde083cc23b60ad6696958909849):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200114-1688019/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1206-g3d98dae
		Tests run against pgx v4.2.1
		116 Total Tests Run
		62 tests passed
		54 tests failed
		0 tests skipped
		2 tests ignored
		1 test passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		--- PASS: v4.TestArrayDecoding - https://github.com/cockroachdb/cockroach/issues/26925 (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1688019&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690462&tab=buildLog) on [release-19.2@99a4ab5a34b310338098a6aafb6621920fd9d4cc](https://github.com/cockroachdb/cockroach/commits/99a4ab5a34b310338098a6aafb6621920fd9d4cc):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200115-1690462/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-76-g99a4ab5
		Tests run against pgx v4.2.1
		116 Total Tests Run
		62 tests passed
		54 tests failed
		0 tests skipped
		2 tests ignored
		1 test passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		--- PASS: v4.TestArrayDecoding - https://github.com/cockroachdb/cockroach/issues/26925 (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist19_2) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690462&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690426&tab=buildLog) on [master@0046d62b00aa9e74489d1f44125948c9af729e2b](https://github.com/cockroachdb/cockroach/commits/0046d62b00aa9e74489d1f44125948c9af729e2b):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200115-1690426/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1249-g0046d62
		Tests run against pgx v4.2.1
		116 Total Tests Run
		62 tests passed
		54 tests failed
		0 tests skipped
		2 tests ignored
		1 test passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: v4.TestArrayDecoding - https://github.com/cockroachdb/cockroach/issues/26925 (unexpected)
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690426&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693632&tab=buildLog) on [release-19.2@297c398803afd80276db8dc78b3f7294822c87a8](https://github.com/cockroachdb/cockroach/commits/297c398803afd80276db8dc78b3f7294822c87a8):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200116-1693632/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-83-g297c398
		Tests run against pgx v4.2.1
		116 Total Tests Run
		62 tests passed
		54 tests failed
		0 tests skipped
		2 tests ignored
		1 test passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		--- PASS: v4.TestArrayDecoding - https://github.com/cockroachdb/cockroach/issues/26925 (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist19_2) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693632&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693614&tab=buildLog) on [master@e2c2be6cbef07c7f2ee6341816f81f72a206c32a](https://github.com/cockroachdb/cockroach/commits/e2c2be6cbef07c7f2ee6341816f81f72a206c32a):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200116-1693614/pgx/run_1
	orm_helpers.go:181,orm_helpers.go:111,pgx.go:111,pgx.go:122,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1277-g379fc8f
		Tests run against pgx v4.2.1
		116 Total Tests Run
		62 tests passed
		54 tests failed
		0 tests skipped
		2 tests ignored
		1 test passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestLargeObjectsPreferSimpleProtocol - unknown (unexpected)
		--- PASS: v4.TestArrayDecoding - https://github.com/cockroachdb/cockroach/issues/26925 (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blacklist (pgxBlacklist20_1) is available in the artifacts' pgx log
```

<details><summary>Repro</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693614&tab=artifacts#/pgx)
```
make stressrace TESTS=pgx PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/pgx_blacklist.go
+++ b/pkg/cmd/roachtest/pgx_blacklist.go
@@ -20,7 +20,6 @@ var pgxBlacklists = blacklistsForVersion{
 // in the test log.
 var pgxBlackList20_1 = blacklist{
 	"v4.Example_CustomType":                                                   "27796",
-	"v4.TestArrayDecoding":                                                    "26925",
 	"v4.TestConnBeginBatchDeferredError":                                      "31632",
 	"v4.TestConnCopyFromCopyFromSourceErrorEnd":                               "5807",
 	"v4.TestConnCopyFromCopyFromSourceErrorMidway":                            "5807",
@@ -59,6 +58,7 @@ var pgxBlackList20_1 = blacklist{
 	"v4.TestInsertTimestampArray":                                             "5807",
 	"v4.TestLargeObjects":                                                     "26725",
 	"v4.TestLargeObjectsMultipleTransactions":                                 "26725",
+	"v4.TestLargeObjectsPreferSimpleProtocol":                                 "26725",
 	"v4.TestListenNotify":                                                     "41522",
 	"v4.TestListenNotifySelfNotification":                                     "41522",
 	"v4.TestListenNotifyWhileBusyIsSafe":                                      "41522",
@@ -82,7 +82,6 @@ var pgxIgnoreList20_1 = blacklist{
 
 var pgxBlackList19_2 = blacklist{
 	"v4.Example_CustomType":                                                   "27796",
-	"v4.TestArrayDecoding":                                                    "26925",
 	"v4.TestConnBeginBatchDeferredError":                                      "31632",
 	"v4.TestConnCopyFromCopyFromSourceErrorEnd":                               "5807",
 	"v4.TestConnCopyFromCopyFromSourceErrorMidway":                            "5807",
@@ -119,8 +118,9 @@ var pgxBlackList19_2 = blacklist{
 	"v4.TestInetCIDRTranscodeWithJustIP":                                      "18846",
 	"v4.TestInsertBoolArray":                                                  "5807",
 	"v4.TestInsertTimestampArray":                                             "5807",
-	"v4.TestLargeObjects":                                                     "42626",
-	"v4.TestLargeObjectsMultipleTransactions":                                 "42626",
+	"v4.TestLargeObjects":                                                     "26725",
+	"v4.TestLargeObjectsMultipleTransactions":                                 "26725",
+	"v4.TestLargeObjectsPreferSimpleProtocol":                                 "26725",
 	"v4.TestListenNotify":                                                     "41522",
 	"v4.TestListenNotifySelfNotification":                                     "41522",
 	"v4.TestListenNotifyWhileBusyIsSafe":                                      "41522",

